Cargo.toml

```diff
@@ -39,7 +39,7 @@ bytes = "0.5.3"
 hex = "0.3.1"
 fern = { version = "0.5.9", features = ["colored", "syslog-4"] }
 clap = { version = "2.33.0", features = ["yaml"] }
-warp = { version = "0.2.0", features = ["tls"] }
+warp = { git = "https://github.com/seanmonstar/warp", features = ["tls", "compression"] }
```

Comment:
Didn't notice this yesterday, but you probably want to add a revision to this. At least that way Lucid doesn't break when the warp `master` does, and since `master` doesn't have any specific version there are no guarantees about API compatibility. 

`warp = { git = "https://github.com/seanmonstar/warp", rev  ="63c506a", features = ["tls", "compression"] }`

Reply:
Yeah but you know we will wait I think for the release of compression to does not use the repo directly

Reply:
I agree, waiting until it's released on crates.io is best. Depending on a specific revision is just to avoid things breaking randomly if/when the `warp` team changes any part of the compression API.
It's just a recommendation though, feel free to depend on the head of `master` if you want. I think it won't cause problems as long as you're the only one working on the `http-compression` branch.

Reply:
Alright